staff/henry-novoa/cv-styled/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Curriculum vitae</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="box info">
```

Comment:
You could try to add a photograph to the right of the "box info"

Reply:
Will add if I have time, adding an image requires also  reconfiguring the structure